### ISSUE TITLE
'Include' filtering with values that are a part of the unique values of that field throws error

### ISSUE THREAD

**Issue by turnerniles** [pat310/quick-pivot#46]
```
const dataArray = [
 ['name', 'gender', 'house', 'age'],
 ['Jon', 'm', 'Stark', 14],
 ['Arya', 'f', 'Stark', 10],
 ['Cersei', 'f', 'Baratheon', 38],
 ['Tywin', 'm', 'Lannister', 67],
 ['Tyrion', 'm', 'Lannister', 34],
 ['Joffrey', 'm', 'Baratheon', 18],
 ['Bran', 'm', 'Stark', 8],
 ['Jaime', 'm', 'Lannister', 32],
 ['Sansa', 'f', 'Stark', 12]
];

const pivot = new pivot(dataArray, ['gender'], [], 'age', 'count');

pivot.filter('gender', ['dogman'], 'include')
```
![image](https://cloud.githubusercontent.com/assets/8146241/24893508/1f8b4296-1e52-11e7-9bb0-f0ba769b23f5.png)

**Comment by turnerniles** [comment]
Similarly 

```
pivot.filter('dogman', ['dogman'], 'include')
```
Throws the same error.

I think for both of these scenarios, it should return an 'empty' pivot (headers, no data). Exclude works in both scenarios.

**Comment by turnerniles** [comment]
Well I'm not sure about the second scenario. Perhaps `pivot.filter('dogman', ['dogman'], 'include')` should throw an error

**Comment by pat310** [comment]
Closed in #49

**Comment by pat310** [comment]
@turnerniles I think #49 fixes this but please reopen if it's still an issue!
